pandas/core/internals.py

```diff
@@ -1854,11 +1859,40 @@ class ExtensionBlock(NonConsolidatableMixIn, Block):
 
     ExtensionArrays are limited to 1-D.
     """
+    is_extension = True
+
+    def __init__(self, values, placement, ndim=None):
+        values = self._maybe_coerce_values(values)
+        super(ExtensionBlock, self).__init__(values, placement, ndim)
+
+    def _maybe_coerce_values(self, values):
```

Comment:
is this called anywhere else? (e.g. IOW I *think* can eliminate some code if maybe this was defined in the superclass)

Reply:
No, this is only called in `__init__`, and was made into a separate method on your question. It's not defined in a superclass, so cannot be removed.

Reply:
https://github.com/pandas-dev/pandas/issues/19492

Eventually Block.__init__ will call `self._maybe_coerce_values` and we can remove every subclasses's __init__. But there's some other work to be done first.

Reply:
ok